MSAL/test/unit/MSALInteractiveRequestTests.m

```diff
@@ -44,23 +43,42 @@
 #import "MSIDTestURLSession.h"
 #import "MSIDTestURLResponse.h"
 #import "NSDictionary+MSIDTestUtil.h"
+#import "MSIDSharedTokenCache.h"
+#import "MSIDDefaultTokenCacheAccessor.h"
+#import "MSIDKeychainTokenCache.h"
+#import "MSIDKeychainTokenCache+MSIDTestsUtil.h"
+#import "MSIDMacTokenCache.h"
 
 @interface MSALInteractiveRequestTests : MSALTestCase
 
+@property (nonatomic) MSIDSharedTokenCache *tokenCache;
+@property (nonatomic) MSIDDefaultTokenCacheAccessor *tokenCacheAccessor;
+
 @end
 
 @implementation MSALInteractiveRequestTests
 
-- (void)setUp {
+- (void)setUp
+{
     [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
+    
+#if TARGET_OS_IPHONE
+    [MSIDKeychainTokenCache reset];
```

Comment:
can we have a test utility to create correct cache? I feel this code is repeating.